pkg/registry/apis/alerting/notifications/register.go

```diff
@@ -100,6 +101,26 @@ func (t NotificationsAPIBuilder) GetAPIRoutes() *builder.APIRoutes {
 	return nil
 }
 
+// PostProcessOpenAPI is a hook to alter OpenAPI3 specification of the API server.
+func (b *NotificationsAPIBuilder) PostProcessOpenAPI(oas *spec3.OpenAPI) (*spec3.OpenAPI, error) {
```

Comment:
We're mixing methods on pointer and value receivers, let's pick one. The other builders are using pointer receivers.